containers/glassfish/jersey-gf-ejb/src/main/java/org/glassfish/jersey/gf/ejb/internal/EjbComponentProvider.java

```diff
@@ -347,6 +368,11 @@ private void logLookupException(final Method method, final Class<?> component, C
                 allLocalOrRemoteIfaces.add(i);
             }
         }
+        if (allLocalOrRemoteIfaces.isEmpty()) {
+            for (Class<?> i : resourceClass.getInterfaces()) {
+                allLocalOrRemoteIfaces.add(i);
+            }
+        }
```

Comment:
EJB Spec, paragraph 4.9.7:

`java.io.Serializable`, `java.io.Externalizable`  and  the  interfaces  defined  by  the  `javax.ejb`  package  are  excluded  when  determining  whether  the  bean class has business interfaces

Reply:
I'm not sure what level of correctness is needed for the method, but if `resourceClass` have `Local` or `Remote` annotation present, the algorithm should look no further. And of course any of these can be overriden in deployment descriptor. But my feeling is that we're good with just excluding `javax.ejb`.

Reply:
That's why is there the `if (allLocalOrRemoteIfaces.isEmpty())` { part

Reply:
The exclusion of javax.ejb, etc. Ok, I will implement it.